.travis.yml

```diff
@@ -44,7 +44,7 @@ matrix:
     - env: DISTRIB="conda" PYTHON_VERSION="*"
            NUMPY_VERSION="*" SCIPY_VERSION="*"
            SCIKIT_LEARN_VERSION="*" MATPLOTLIB_VERSION="*" COVERAGE="true"
-           PANDAS_VERSION="*"
+           PANDAS_VERSION="*" BOTO3=true
```

Comment:
By default, BOTO3='' ?

Reply:
This is the only build where BOTO3 dependent function is tested. There doesn't need to be a BOTO3 env var in other builds.